src/main/scala/edu/berkeley/ce/sparkrocks/SeedJointSelector.scala

```diff
@@ -74,7 +74,7 @@ object SeedJointSelector {
     } else if (jointSets.head(0).shape.isEmpty) {
       val (seedJoints, leftOverJoints) = searchJointSets(jointSets, inputVolume, numPartitions, jointSetSpan)
       val partitioning = seedJoints.foldLeft(Seq(inputVolume)) { (currentBlocks, joint) =>
-        currentBlocks.flatMap(_.cut(joint))
+        currentBlocks.flatMap(_.cut(joint, minSize = 0.0, maxAspectRatio = Double.PositiveInfinity))
```

Comment:
Cool!

What this means is that you  now don't have to specify the `minSize` and `maxAspectRatio` explicitly anymore if you're sticking to the defaults. So I think there are a few places where you can those arguments in the code.